src/client/eppo-client.ts

```diff
@@ -540,17 +540,17 @@ export default class EppoClient implements IEppoClient {
     }
 
     // assignment logger may be null while waiting for initialization
-    if (this.assignmentLogger == null) {
+    if (!this.assignmentLogger) {
```

Comment:
❤️  I too like taking advantage of truthiness and falsiness

Reply:
less code -> less bugs 🗡️